### PR TITLE
Permissions readme

### LINKED ISSUE #8: Question: Does this action work with pull requests from forks?
Hi, I'm looking for an app or action to label pull requests with a size based on LOC. This action looks perfect, but another similar action I tried did not work on PRs created from forks (which is the majority of PRs in our project). Can you confirm if this action will work on PRs from forks?

### LINKED ISSUE #47: Document required permissions
It took me a bit of time to figure out which [permissions](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview) this actions needs.

In my case, adding the following worked. Maybe that should be added to the README.

```yml
jobs:
  labeler:
    ...
    permissions:
      issues: write
      pull-requests: write
    steps:
      ...
```

### PR DESCRIPTION
**Pull request by Achllle**
## What type of PR is this? (check all applicable)

- [x] Documentation Update

## Description

- Adds the required permissions to the README for this to work at all.
- Notes for those using forks

## Notes to Reviewer

I've made some other minor improvements, including specifying the full version in the README's action.yml example. With the current `v1`, some of the features described in the same readme do not work. Happy to replace that with a note specifying people should consider using the latest version.

## Link to issues addressed

- closes #47
- closes #8

### CODE REVIEW

**Comment by JavierCane** [comment]
Nice one @Achllle! Thanks a lot for the contribution 😊

I've given some feedback and suggested changes. Please review them when you get a chance 🙌

**Review by JavierCane — CHANGES_REQUESTED** [review]
Nice one @Achllle! Thanks a lot for the contribution 😊

I've given some feedback and suggested changes. Please review them when you get a chance 🙌

File: README.md [hunk 1]
```diff
@@ -36,10 +36,14 @@ on: [pull_request]
...
   labeler:
+    permissions:
+        pull-requests: write
+        contents: read
+        issues: write
     runs-on: ubuntu-latest
     name: Label the PR size
     steps:
-      - uses: codelytv/pr-size-labeler@v1
+      - uses: codelytv/pr-size-labeler@v1.10.0
```
[the remark this answers is not in the document]
Comment (Achllle) [reply README.md:46]: Sounds good, I'll revert my commit.

File: README.md [hunk 2]
```diff
@@ -60,6 +64,8 @@ jobs:
           files_to_ignore: ''
 ```
 
+**note**: When using forks and where you don't want any PR to be able to execute code, replace `on: [pull_request]` with `on: [pull_request_target]` (see [GitHub docs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target)).
```
[the remark this answers is not in the document]
Comment (Achllle) [reply README.md:67]: I should've seen this, thanks. Done manually since your suggestion contained a typo.

**Review by JavierCane — APPROVED** [review]
Thanks for reviewing the suggestions. Merging! 😊

**Pull request merged**